breathecode/mentorship/receivers.py

```diff
@@ -11,3 +14,10 @@
 #     if instance.status == 'COMPLETED':
 #         logger.debug('Procesing mentoring session completition')
 #         send_mentorship_session_survey.delay(instance.id)
+
+
+@receiver(mentor_profile_saved, sender=MentorProfile)
```

Comment:
deleting a mentor should check if the `Mentor` group should be removed from the user fir there is not other mentorprofile for that user.